docs/source/getting_started.rst

```diff
@@ -0,0 +1,21 @@
+Getting Started
+===============
+
+The Data Resource Generator (DRG) is a database and API as configuration application.
+
+Terms
```

Comment:
How about "Definitions"? or "Terms to Know"? I thought this meant "Terms of Use" at first, since it appears in the context of a product document.